trx_examples/targeting/loopback-delay-estimation/hdlworkflow.m

```diff
@@ -0,0 +1,189 @@
+%--------------------------------------------------------------------------
+% HDL Workflow Script
+% Generated with MATLAB 9.7 (R2019b) at 16:00:44 on 06/12/2019
+% This script was generated using the following parameter values:
+%     Filename  : 'C:\git\ad9361_loopbackdelay_estimation\hdlworkflow.m'
+%     Overwrite : true
+%     Comments  : true
+%     Headers   : true
+%     DUT       : 'loopback_delay_estimation/HDL_DUT'
+% To view changes after modifying the workflow, run the following command:
+% >> hWC.export('DUT','loopback_delay_estimation/HDL_DUT');
+%--------------------------------------------------------------------------
+
+%% Load the Model
+load_system('loopback_delay_estimation');
+
+%% Restore the Model to default HDL parameters
+%hdlrestoreparams('loopback_delay_estimation/HDL_DUT');
+
+%% Prepend Xilinx Vivado path to the system path
+hdlsetuptoolpath('ToolName','Xilinx Vivado','ToolPath','C:\Xilinx\Vivado\2018.2\bin\vivado.bat');
```

Comment:
CI fails on this line since this is windows specific. The test harness will automatically add vivado so you can remove this line.

Reply:
Removed this line.